.github/checkgroup.yml

```diff
@@ -0,0 +1,134 @@
+custom_service_name: "Lightning CI required checker"
+subprojects:
+  - id: "pytorch_lightning"
+    paths:
+      - ".circleci/**"
+      - ".azure/**"
+      - "docs/source-pytorch/**"
+      # examples don't need to be added because they aren't used in CI, but these are
+      - "examples/test_pl_examples.py"
+      - "examples/run_ddp_examples.sh"
+      - "examples/run_pl_examples.sh"
+      - "requirements/pytorch/**"
+      - "src/pytorch_lightning/**"
+      - "tests/tests_pytorch/**"
+      - "setup.cfg"  # includes pytest config
+      - "pyproject.toml"  # includes mypy config
+      - ".readthedocs.yml"
+      - ".github/workflows/_check-schema.yml"
+      - ".github/workflows/ci-schema.yml"
```

Comment:
I would keep this as always required, regardless job, they are also 1min or so

Reply:
Sure